scripts/utils/update_package.py

```diff
@@ -37,7 +37,10 @@ def get_latest_version(org, project, version):
         print(f"GitHub API response not ok: {response.status_code}")
         return None
     latest_version = response.json()["tag_name"]
-    return latest_version
+    if latest_version.startswith('v'):
```

Comment:
@naacbin introduced the support for packages whose version is preceded by `v` in https://github.com/mandiant/VM-Packages/pull/1067, but it seems the `v` was only taken into account when matching the url in the file and not to get the latest version, so it seems like the fix has never worked.

What is really getting me confused is that I see upx being correctly updated by our bot in https://github.com/mandiant/VM-Packages/commit/acec181a0574c4c85f0de99d48f875093b3e23a7 before https://github.com/mandiant/VM-Packages/pull/1067, how is this possible? :female_detective:

Reply:
I still don't understand how it could work in acec181, but I think the change should fix the issue, so I think we can merge it. But I would still be interested in understanding how this broke, so please if someone (maybe @naacbin)  has any clue let me know!